### ISSUE TITLE
Issue with skip-existing and pinning

### ISSUE THREAD

**Issue by johanneskoester** [bioconda/bioconda-recipes#1395]
Hi @bioconda/all,
I recently activated the pinning of certain dependencies (so far boost). While this is properly supported by conda, there seems to be a bug in the conda code that detects whether a package is already uploaded. Instead of being able to skip such packages, conda currently wants to rebuild them over and over.

I will look into this issue now and submit a PR to Continuum ASAP. Please don't merge anything into master until this is solved.

Thanks!

**Comment by johanneskoester** [comment]
Ok, I fixed the issue in our fork of conda-build. I modified our build scripts such that our fork is temporarily used. The changes are summarized in [this PR](https://github.com/conda/conda-build/pull/897). Let's hope they merge it soon.

Feel free to merge into master again :-)!
